### PR TITLE
Support sending to M account

### PR DESCRIPTION
**Pull request by quietbits**

### CODE REVIEW

**Review by piyalbasu — APPROVED** [review]

File: src/helpers/isValidMAccount.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const isValidMAccount = (value: string) =>
+  value.startsWith("M") && value.length === 69;
```
Comment (Shaptic) [inline src/helpers/isValidMAccount.ts:2]: Why not use the SDK's validation methods? e.g. `try { decodeAddressToMuxedAccount(value, true) }`

Comment (quietbits) [reply src/helpers/isValidMAccount.ts:2]: We use `StrKey.isValidEd25519PublicKey` to validate G-address, and `StrKey` didn't show any options for M-address in VS Code. I have to admit that I didn't look deeper into that at the time. 😊

It looks like we have [`isValidMed25519PublicKey`](https://stellar.github.io/js-stellar-base/StrKey.html#.isValidMed25519PublicKey) on `StrKey`, but it's missing in [types](https://github.com/stellar/js-stellar-base/blob/master/types/index.d.ts#L738). Should we use that to validate M-addresses to stay consistent once it's available? @Shaptic

Comment (Shaptic) [reply src/helpers/isValidMAccount.ts:2]: Good point, hard to expect you to use a validation method I never exported for use! :wink: I'll make an issue to track it for myself, and I'll let you know once that gets exposed.

**Comment by shanzzam** [comment]
Tested on my Trezor with their most recent update to suite (v 21.7.1) and device firmware (2.4.1) 
I'm seeing the correct and expected sending error when attempting to send to an M address.

For the purposes of this work, we can merge this PR.
I'll open another issue to update the error messaging users see with a failed transaction.

**Pull request merged**